src/excmds.ts

```diff
@@ -2352,6 +2353,21 @@ export function autocmd(event: string, url: string, ...excmd: string[]) {
     config.set("autocmds", event, url, excmd.join(" "))
 }
 
+/** Automatically open a domain and all its subdomains in a specified container.
+  
+ @param domain The domain which will trigger the autoContain directive. Includes all subdomains.
```

Comment:
As I mention in the other file, please be more explicit about how the domain is matched.